package.json

```diff
@@ -9,16 +9,23 @@
     "cypress:run": "cypress run",
     "start": "next start",
     "test": "jest",
-    "lint": "next lint"
+    "eslint": "next lint",
+    "stylelint": "stylelint '**/*.scss'",
+    "lint": "yarn eslint && yarn stylelint"
   },
   "dependencies": {
+    "@chakra-ui/react": "2.0.2",
```

Comment:
Should we move these new packages into `devDependencies` instead?

Reply:
@hoangmirs These packages are used for making UI components https://chakra-ui.com/guides/getting-started/nextjs-guide#installation.

We should place theme here 🖖